src/Microsoft.AspNet.SignalR.Core/Infrastructure/DispatchingTaskCompletionSource.cs

```diff
@@ -46,9 +41,17 @@ public void TrySetException(Exception exception)
             TaskAsyncHelper.Dispatch(() => _tcs.TrySetException(exception));
         }
 
-        public void TrySetException(IEnumerable<Exception> exceptions)
+        public void SetUnwrappedException(Exception e)
```

Comment:
It wasn't changed in this PR, but TrySetUnwrappedException also doesn't dispatch. I think they both should.